src/group_theory/quotient_group.lean

```diff
@@ -291,4 +299,55 @@ have φ_surjective : function.surjective φ := λ x, x.induction_on' $
 
 end snd_isomorphism_thm
 
+namespace third_iso_thm
+
+variables (M : subgroup G) [nM : M.normal]
+
+include nM nN
+
+@[to_additive quotient_add_group.map_normal]
```

Comment:
I'm surprised you have to tell `to_additive` the target lemma name

Reply:
Me too, but that is what the `to_additive` told me and what is done in the rest of this file too. I guess we'd have to register that `quotient_group` additivizes to `quotient_add_group` somewhere?